dev.py

```diff
@@ -46,6 +46,26 @@ def cmd_all_npm_build(args):
         frontend_dir = next(project_dir.glob("*/frontend/"))
         run_verbose(["npm", "run", "build"], cwd=str(frontend_dir))
 
+
+def cmd_all_install_python_deps(args):
+    for project_dir in EXAMPLE_DIRECTORIES + TEMPLATE_DIRECTORIES:
+        run_verbose(["pip", "install", "-e", ".[devel]"], cwd=str(project_dir))
```

Comment:
Why is `[devel]` needed?

Reply:
Because we also want to install dependencies that are needed in the development environment. See changes in `setup.py` file.

The `devel` word is a convention commonly used in Python world. For example, see: https://airflow.apache.org/docs/apache-airflow/stable/extra-packages-ref.html

Reply:
Because i've put the dependencies in `extras_require` in setuptools and named them `devel` according to the convention used in e.g. apache airflow, so I had to add it to the args here.